README.md

```diff
@@ -110,17 +119,12 @@ To run the project, execute the following command:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FUTURE FEATURES -->
-
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **[Add a Calculator]**
-- [ ] **[Style the UI]**
+> ** Add a Calculator ** > ** Style the UI **
```

Comment:
- [optional] kindly ensure the future features are related to your app.. Here, you are talking of a calculator.